task_14/js/task5.js

```diff
@@ -0,0 +1,7 @@
+let name = prompt('введите Ваше имя');
+let clearName = name.trim();
+if (clearName == 0 || clearName == ' '){
```

Comment:
clearName == 0
Are you sure?)
I think 
`if (clearName == ''){` is more understandable